src/30_Advanced/Using_the_Google_AMP_Cache.html

```diff
@@ -0,0 +1,67 @@
+<!--
+  #### Introduction
+
+  The Google AMP Cache stores valid AMP pages and provides a consistently fast access to these.
+
+-->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <style amp-custom>
+  .warning {
+    background-color: #F8BBD0;
+  }
+    </style>
+</head>
+<body>
+  <!-- #### AMP Cache URL Format -->
+  <!--
+    Prepend `cdn.ampproject.org/c/` to AMP HTTP Urls. 
+  -->
+    <a href="http://www.bbc.co.uk/news/amp/35838735">Link with AMP HTTP URL</a>
+    <a href="https://cdn.ampproject.org/c/www.bbc.co.uk/news/amp/35838735">Link with AMP HTTP URL accessing the cache</a>
+
+  <!--
+    Prepend `cdn.ampproject.org/c/s` to AMP HTTPS Urls. 
+
+  -->
+    <a href="https://ampbyexample.com/components/amp-video/">Link with AMP HTTPS URL</a>
+    <a href="https://cdn.ampproject.org/c/s/ampbyexample.com/components/amp-video/">Link with AMP HTTPS URL accessing the cache</a>
+    
+    <!--
+      If URLs you are using are using GET parameters for implementing specific logics, you can still specify them in the new URL created for using the AMP cache.
+    -->
+
+    <a href="https://cdn.ampproject.org/c/s/ampbyexample.com/g?value=Hello%20World">Link containing a GET argument.</a>
+    
+    <!-- #### Redirect & Error Handling -->
+    <!--
+      If your server is redirecting a particular URL to another URL, this will still work when using AMP cache.
+    -->
+    <a href="https://cdn.ampproject.org/c/s/ampbyexample.com/amp-img.html">Link which will result in a redirect to https://ampbyexample.com/components/amp-img/.</a>
+    <p class="card warning">If you move the location of the AMP Files on your server, make sure to set up a redirect from the old location to the new one. </p>
```

Comment:
can you add a css class for warning. So my comment for the previous commit for a suggested color.